packages/radix-ui-themes/changelog.md

```diff
@@ -159,10 +161,10 @@
   - Add `color` and `highContrast` props to `TabsList`
   - Add margin props `TabsList` and `TabsContent`
   - Renamed the letter/word spacing CSS variables in `.radix-themes` so that it supports both `Tabs` and `TabNav` components. Note that this is a backwards compatible change, as the old variables are still supported.
-    - `--tabs-trigger-active-letter-spacing` -> `--tab-active-letter-spacing`
-    - `--tabs-trigger-active-word-spacing` -> `--tab-active-word-spacing`
-    - `--tabs-trigger-inactive-letter-spacing` -> `--tab-inactive-letter-spacing`
-    - `--tabs-trigger-inactive-word-spacing` -> `--tab-inactive-word-spacing`
+    - `--tabs-trigger-active-letter-spacing` → `--tab-active-letter-spacing`
```

Comment:
Above it says:
> Note that this is a backwards compatible change, as the old variables are still supported.

Not true anymore I believe, we got rid of the backwards compatibility?

Reply:
Looks like it still exists
<img width="878" alt="CleanShot 2024-03-22 at 14 38 30@2x" src="https://github.com/radix-ui/themes/assets/1539897/4c37bd5e-cc68-4610-a28f-0d37c49a8f44">